models/gas/optimism/gas_optimism_fees_traces_legacy.sql

```diff
@@ -1,13 +1,12 @@
 {{ config(
-	tags=['legacy'],
-	
-    schema = 'gas_optimism',
-    alias = alias('fees_traces', legacy_model=True),
-    partition_by = ['block_date'],
-    materialized = 'incremental',
-    file_format = 'delta',
-    incremental_strategy = 'merge',
-    unique_key = ['tx_hash', 'trace'],
+     tags=['legacy'],
+     schema = 'gas_optimism',
+     alias = alias('fees_traces', legacy_model=True),
+     partition_by = ['block_date'],
+     materialized = 'incremental',
+     file_format = 'delta',
+     incremental_strategy = 'merge',
```

Comment:
Since we are gonna modify these anyway we could also add `incremental_predicates = ['DBT_INTERNAL_DEST.block_time >= date_trunc(\'day\', now() - interval \'1\' week)'],` here to make it less heavy. It worked pretty well for Polygon.

Reply:
testing it now

Reply:
it works see below, it's on the polygon version for databricks